snyk-monitor/templates/deployment.yaml

```diff
@@ -44,11 +44,11 @@ spec:
               - matchExpressions:
                 - key: "kubernetes.io/arch"
                   operator: In
-                  values: ["amd64"]
+                  values: ["amd64", "arm64"]
```

Comment:
Nit: Would it worth to update that in `snyk-operator-certified/helm-charts/snyk-monitor/templates/deployment.yaml` too?

Reply:
updated